tests/testthat/test-rng.R

```diff
@@ -265,3 +265,96 @@ test_that("get state", {
   expect_identical(s3[seq_len(32)], s1)
   expect_identical(s3[-seq_len(32)], rng2$jump()$state())
 })
+
+
+test_that("initialise single rng with binary state", {
+  seed <- 42
```

Comment:
If you do allow negative numbers, add a test here

Reply:
done